valkyrie/lib/valkyrie/persistence/postgres/persister.rb

```diff
@@ -29,5 +29,10 @@ def delete(resource:)
       orm_object.delete
       resource
     end
+
+    def wipe!
+      DatabaseCleaner.clean
```

Comment:
This'll only work in specs. `resource_factory.orm_class.delete_all` should do it, though.

Reply:
haha i thought that looked suspicious! thanks.